frontend/modules/common/Api.js

```diff
@@ -0,0 +1,56 @@
+import request from 'superagent';
```

Comment:
What, you mean one file for recipes, one for the grocery list, etc? Yeah that makes sense, but I think that's probably better to make as a separate issue.